src/common-components/PasswordField.jsx

```diff
@@ -138,7 +138,7 @@ const PasswordField = (props) => {
       {props.errorMessage !== '' && (
         <Form.Control.Feedback key="error" className="form-text-size" hasIcon={false} feedback-for={props.name} type="invalid">
           {props.errorMessage}
-          <span className="sr-only">{formatMessage(messages['password.sr.only.helping.text'])}</span>
+          {props.showScreenReaderText && <span className="sr-only">{formatMessage(messages['password.sr.only.helping.text'])}</span>}
```

Comment:
We provide a tooltip on the registration form, but it is not accessible for screen readers. Instead, for screen readers, we utilize an additional span as a helper text.
```
Password must contain at least 8 characters, at least one letter, and at least one number
```

Reply:
i understand but my question was more around "Why a prop is needed?"

Reply:
This helper text serves as an alternative for screen readers. Since the visual helper toolkit is not displayed on the LoginPage, we don't need to show this span too. It is important to ensure that the functionality aligns for both screen reader users and other individuals.

|Registration|Login|
|--------|--------|
<img width="651" alt="Screenshot 2024-01-22 at 4 49 35 PM" src="https://github.com/openedx/frontend-app-authn/assets/40633976/cdc6c563-6fa2-4f14-9238-52adc9fc3dcd">|<img width="651" alt="Screenshot 2024-01-22 at 4 49 44 PM" src="https://github.com/openedx/frontend-app-authn/assets/40633976/143dc24f-d0e9-469b-acb3-a3a2e94d9c67">

Reply:
Makes sense, thankyou